services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala

```diff
@@ -44,16 +44,33 @@ abstract class WorkbenchHealthMonitorServiceActor(val serviceConfig: Config, glo
   }
 
   protected lazy val Gcs = MonitoredSubsystem("GCS", () => checkGcs())
+  protected lazy val CarboniterGcsAccess = MonitoredSubsystem("CarboniterGcsAccess", () => checkGcsAccess(carboniterBucketName, gcsObjectToCheckCarboniterAccess, carboniterGoogleAuth))
   protected lazy val PapiSubsystems = papiBackendConfigurations map papiMonitoredSubsystem
 
   lazy val googleConfig = GoogleConfiguration(globalConfig)
 
   lazy val googleAuthName = serviceConfig.as[Option[String]]("google-auth-name").getOrElse("application-default")
+  lazy val googleAuth = getGoogleAuthConfigurationOrFail(googleAuthName)
 
-  lazy val googleAuth = googleConfig.auth(googleAuthName) match {
-    case Valid(a) => a
-    case Invalid(e) => throw new IllegalArgumentException("Unable to configure WorkbenchHealthMonitor: " + e.toList.mkString(", "))
-  }
+  private lazy val carboniterAuthNameConfigPath = "services.MetadataService.config.carbonite-metadata-service.filesystems.gcs.auth"
+  lazy val carboniterAuthName = getGlobalConfigValueOrFail(carboniterAuthNameConfigPath)
+  lazy val carboniterGoogleAuth = getGoogleAuthConfigurationOrFail(carboniterAuthName)
+
+  private lazy val carboniterBucketNameConfigPath = "services.MetadataService.config.carbonite-metadata-service.bucket"
+  lazy val carboniterBucketName = getGlobalConfigValueOrFail(carboniterBucketNameConfigPath)
+  lazy val gcsObjectToCheckCarboniterAccess = serviceConfig.as[String]("gcs-object-to-check-carboniter-access")
+
+  private def getGlobalConfigValueOrFail(configPath: String): String =
+    globalConfig.as[Option[String]](configPath) match {
+      case Some(value) => value
+      case None => throw new IllegalArgumentException(s"Unable to configure WorkbenchHealthMonitor: path $configPath does not exist in the Cromwell configuration file")
```

Comment:
If we go down this unhappy path what state will Cromwell be in?

Reply:
It will be restarting this actor until The End Of Time, but this also would happen in case of failure in the existing code, e.g.: https://github.com/broadinstitute/cromwell/blob/bb91499e3ccb8c6a0eb7bee61e4e8ff4ec6064e7/services/src/main/scala/cromwell/services/healthmonitor/impl/workbench/WorkbenchHealthMonitorServiceActor.scala#L55

So I think this is a bug and I'm going to create a separate ticket to address it.

Reply:
https://broadworkbench.atlassian.net/browse/BA-6343